miqa/settings.py

```diff
@@ -46,7 +46,17 @@ class MiqaMixin(ConfigMixin):
     # Override default signup sheet to ask new users for first and last name
     ACCOUNT_FORMS = {'signup': 'miqa.core.rest.accounts.AccountSignupForm'}
 
-    CELERY_BEAT_SCHEDULE = {}
+    @property
+    def CELERY_BEAT_SCHEDULE(self):
+        if self.DEMO_MODE:
+            return {
+                'reset-demo': {
+                    'task': 'miqa.core.tasks.reset_demo',
+                    'schedule': timedelta(days=1),
```

Comment:
This also changes from `crontab` to `timedelta` -- is that a necessary part of this fix?

Reply:
I couldn't get the task to run with crontab; I only observed it working with timedelta